vars.yml

```diff
@@ -36,6 +36,8 @@ bbb_meteor:
         name: High definition
```

Comment:
Let's hide this profile, set hidden:true

Reply:
requesting also the following:
set low to 100
medium to 150
leave high alone
decrease low-u8 to 90
decrease low-u12 to 80